targets/mediatek-filogic

```diff
@@ -50,6 +53,21 @@ device('netgear-wax220', 'netgear_wax220', {
 	factory_ext = '.img',
 })
 
+-- OpenWRT
+device('openwrt-one', 'openwrt_one',{
+	factory = '-factory',
+	factory_ext = '.ubi',
+	sysupgrade_ext = '.itb',
+	extra_images = {
+		{'-nor-bl31-uboot', '-nor-bl31-uboot', '.fip'},
+		{'-nor-factory', '-nor-factory', '.bin'},
+		{'-nor-preloader', '-nor-preloader', '.bin'},
+		{'-snand-bl31-uboot', '-snand-bl31-uboot', '.fip'},
+		{'-snand-factory', '-snand-factory', '.bin'},
+		{'-snand-preloader', '-dnand-preloader', '.bin'}
+	},
+})
```

Comment:
I prefer we skip the bootloader images and refer for installation to the latest OpenWrt release to obtain images from there.

Reply:
Hello @rotanid, should I just stick to the sysugrade image then and remove all other images?

Reply:
I'd be fine with your current state.